### ISSUE TITLE
Support for Fedora 22 broken?

### ISSUE THREAD

**Issue by NLSurfMan** [nodesource/distributions#227]
Although the [frontpage](https://github.com/nodesource/distributions#rpm) says Fedora 22 is supported, I can't get the setup script to work. I have a simple, out-of-the-box installation of Fedora 22.

```bash
[root@XYZ ~]# curl -sL https://rpm.nodesource.com/setup | bash -

## Installing the NodeSource Node.js 0.10 repo...


## Inspecting system...

+ rpm -q --whatprovides redhat-release || rpm -q --whatprovides centos-release || rpm -q --whatprovides cloudlinux-release || rpm -q --whatprovides sl-release
+ uname -m

## Confirming "fc22-x86_64" is supported...

+ curl -sLf -o /dev/null 'https://rpm.nodesource.com/pub_0.10/fc/22/x86_64/nodesource-release-fc22-1.noarch.rpm'

## Your distribution, identified as "fedora-release-22-1.noarch", is not currently supported, please contact NodeSource at https://github.com/nodesource/distributions/issues if you think this is incorrect or would like your distribution to be considered for support

[root@XYZ ~]# 
```

The URL generated doesn't even come close to what is available in [the file listing](https://rpm.nodesource.com/pub_0.10/fc/22/x86_64/)

Output of *uname -a*:
```
Linux pcwj 4.2.8-200.fc22.x86_64 #1 SMP Tue Dec 15 16:50:23 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
```

Output of *cat /etc/fedora-release*:
```
[root@XYZ ~]# cat /etc/fedora-release
Fedora release 22 (Twenty Two)
```

If you want more testing and output, just let me know. Thanks for your effort!

**Comment by chrislea** [comment]
Well, so that was **completely** our fault. Thanks for the catch!

Could you please try it again now? I think the problem should be fixed.

**Comment by NLSurfMan** [comment]
For 0.10 this works. Would you be so kind to do this for 0.12 too? That's the version I was actually after.

**Comment by chrislea** [comment]
Okay @NLSurfMan , this should now be fixed for 0.12 as well.

I would be remiss in not saying that if you are using 0.12 currently, you should *seriously* look at moving to 4.2.x as soon as you can. Very little if anything will need to change in your code, and the number of things that got fixed / better between 0.12 and 4.x is absolutely **HUGE**.

**Comment by NLSurfMan** [comment]
I have installed the 5.x version. That worked like a charm.

I must admit I am not up to speed when it comes to Node versions. I went for 0.12 because I am getting started with EmberJS 2.3.0 and that required it. I was confused about the jump in version number from 0.12 to 4/5, so I played it safe. 

Thanks for the effort!
